api/src/templates/login/password-reset.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#323232">
+    <title>{{ branding.name }}</title>
+    <link href="/login/style.css" media="all" rel="stylesheet">
+    <link rel="shortcut icon" href="/favicon.ico">
+</head>
+<body data-default-locale=" {{ defaultLocale }}" data-translations="{{ translations }}">
+<div class="center">
+    <form id="form" action="/medic/password-reset" method="POST">
+        <img class="logo" src="{{ branding.logo }}" alt="logo" title="{{ branding.name }}">
```

Comment:
One test is to run it in a different config with branding that is not CHT, make sure the logo displayed nicely